doc/Overview/Mvux/Feeds.md

```diff
@@ -0,0 +1,259 @@
+public async ValueTask<CounterValue> CountOne(CancellationToken ct)
+{
+    await Task.Delay(TimeSpan.FromSeconds(1));
+
+    // note that a service does not normally hold data
+    // this example is for demonstration purposes
+    return new CounterValue(++_currentCount);
+}
+```
+
+> [!NOTE]
+> `ValueTask` is interchangeable with `Task`, but `ValueTask` was chosen to be in unity with the `IAsyncEnumerable` interface.
+> A `Task` is easily convertible to `ValueTask` nonetheless.
+
+This is known as a 'pull' method, as we're repeatedly calling the Task when we're looking for new data,
+and the Task returns the value when it's ready, unless it was cancelled using the token (this will be discussed in another tutorial).
+
+Using the `CountOne` method, creating a Feed is as easy as:
+
+```c#
+public IFeed Value => Feed.Async(_myService.CountOne);
+```
+
+`Feed` is a static class that provides Feed factory methods, as well as extension methods for Feeds.  
+As mentioned above, the `Async` method takes a delegate of the a signature returning `ValueTask<T>` where `T` is the returned type, and has a `CancellationToken` parameter.
+
+Should the signature of your method be different, for example if the method returns `Task<T>` (instead of `ValueTask<T>`, or when it a `CancellationToken` parameter is not present, `Feed.Async` can be called as follows:
+
+```c#
+// Service method
+public async Task<CounterValue> CountOne();
+
+// Feed creation
+public IFeed<CounterValue> CurrentCount => Feed.Async(async ct => await _myService.CountOne());
+```
+
+#### From Async Enumerables
+
+In contrast to Tasks which operate as 'pull' methods, the 'push' method is where we call a method and establish some sort of connection with it,
+while it sends new data packets as they become available:
+
+```c#
+public async IAsyncEnumerable<CounterValue> StartCounting([EnumeratorCancellation] CancellationToken ct)
+{
+    while (!ct.IsCancellationRequested)
+    {
+        await Task.Delay(TimeSpan.FromSeconds(1));
+                
+        return new CounterValue(i);
+    }
+}
+```
+
+Referring to the Async Enumerable from the example a Feed can be created in the following way:
+
+```c#
+public async IAsyncEnumerable<CounterValue> StartCounting(CancellationToken ct);
+
+public IFeed<CounterValue> CurrentCount => Feed.AsyncEnumerable(StartCounting);
+```
+
+`CancellationToken`s are essential to enable halting an ongoing async operation.  
+However, if the API you're consuming does not have a `CancellationToken` parameter, you can disregard that incoming `CancellationToken` parameter as following:
+
+```c#
+public IFeed<CounterValue> CurrentCount => Feed.AsyncEnumerable(ct => StartCounting());
+```
+
+> [!NOTE]
+> `Feed` is a static class that provide factory methods that create `IFeed<T>`s, as well as extension methods for `IFeed<T>`.
+
+> [!NOTE]  
+> There are additional ways to load data (e.g. Observables), but most of them are easily convertible to one of the above two.
```

Comment:
Showing conversation from Observable would be useful. In fact an "Advanced" topic would show how to combine MVUX with IObservables

Reply:
Once link will be available will definitely come here. I'll make a note on my personal ToC.